src/fidgets/layout/Grid.tsx

```diff
@@ -363,7 +366,7 @@ const Grid: LayoutFidget<GridLayoutProps> = ({
     <>
       {editorPanelPortal(element)}
 
-      <div className="flex flex-col">
+      <div className="flex flex-col z-10">
```

Comment:
This makes all of the grid editing stuff sit in front of customize styling like we did for Profiles in #154